suseconnect-ng.spec

```diff
@@ -18,7 +18,7 @@
 %global project github.com/SUSE/connect-ng
 
 Name:           suseconnect-ng
-Version:        1.7.0
```

Comment:
Please update to the next minor version

Reply:
addressed in [981b57a](https://github.com/SUSE/connect-ng/pull/212/commits/981b57a88e2a62ce97852f7fa892e8308cbc454c)